research-app/src/App.vue

```diff
@@ -116,6 +116,17 @@
                       collection</a
                     >
                   </li>
+                  <li>
+                    <a
+                      href="#"
+                      v-close-popover
+                      @click="
+                        selectTool('save-state');
+                        showPopover = false;
+                      "
+                      tabindex="0"
+                    ><font-awesome-icon icon="save" /> Save current state</a>
```

Comment:
I like where you're coming from here but I think the average user might not "get" why the URL export is the same as saving the state. I think a more intuitive title for this would be "Create link to current view", or maybe "Create link to current state".